auth/host_session.go

```diff
@@ -140,7 +142,22 @@ func SessionHandler(conn *websocket.Conn) {
 					return
 				}
 			}
-			logic.CheckHostPorts(&result.Host)
+			if slices.Contains(registerMessage.CustomParams, models.HostRegisterCustomParamListenPort) {
```

Comment:
this fields are already on host model, not required to extend the req to have custom params